src/Core/WinSWCore/Configuration/YamlConfiguration.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+
+namespace winsw.Configuration
+{
+    public class YamlConfiguration
```

Comment:
Move these classes to `/src/Core/WinSWCore/Configuration`?

Reply:
Did you mean one file for each class ?